projects/ngx-float-ui-repo/src/main.ts

```diff
@@ -8,5 +8,5 @@ if (NGX_FLOAT_UI_ENVIRONMENT.production) {
     enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(NgxFloatUiAppModule)
+platformBrowserDynamic().bootstrapModule(NgxFloatUiAppModule, {ngZoneEventCoalescing: true})
```

Comment:
Since version 18 [event coalescing](https://blog.angular.dev/angular-v18-is-now-available-e79d5ac0affe) is on by default, so I enabled it for the demo app. I manually tested the demo app both with event coalescing on and off to make sure there are no regressions.